apps/snitch_core/lib/core/tools/helpers/taxonomy.ex

```diff
@@ -55,4 +55,30 @@ defmodule Snitch.Tools.Helper.Taxonomy do
       create_taxon(taxon, root)
     end
   end
+
+  def convert_to_map(taxonomy) do
```

Comment:
IMO it is better to explicitly implement the `Jason.Encoder` protocol for the taxon and taxonomy structs.